ospfd/ospf_vty.c

```diff
@@ -5089,14 +5089,9 @@ DEFUN (show_ip_ospf_neighbor_all,
 	if (ospf) {
 		ret = show_ip_ospf_neighbor_all_common(vty, ospf, json, uj,
 						       use_vrf);
-		if (uj) {
-			vty_out(vty, "%s\n",
-				json_object_to_json_string_ext(
-					json, JSON_C_TO_STRING_PRETTY));
-		}
-	}
-
-	if (uj)
+		if (uj)
+			vty_json(vty, json);
+	} else if (uj)
 		json_object_free(json);
```

Comment:
I think we should call and print an empty JSON in such a case, right? `vty_json_empty()`

Reply:
in fact  only json part is modified